src/main/java/com/indeed/jiraactions/ActionFactory.java

```diff
@@ -74,6 +74,9 @@ public Action create(final Issue issue) throws IOException {
                 .createdDateTimestamp(issue.fields.created.getMillis() / 1000)
                 .lastUpdated(0)
                 .closedDate(0)
+                .timeOriginalEstimate(issue.fields.timeoriginalestimate == null ? 0 : issue.fields.timeoriginalestimate)
```

Comment:
Shouldn't this call something like `issue.initialValue("timeOriginalEstimate")`?

Reply:
@Anusien, do you mean "timeOriginalEstimate" alone or for all three fields? 
My understanding is these two approaches are equivalent, eventually both of them will get values from "Field.java": `issue.fields.timeoriginalestimate` gets the value directly, while `issue.initialValue()` checks `getStringValue()` at https://github.com/indeedeng/imhotep-builder-jira/blob/master/src/main/java/com/indeed/jiraactions/api/response/issue/fields/Field.java#L61 and we need to add a line for each field. 
But you know it better, so I can change it to the second way if it's better. Please let me know. Thanks.

Reply:
All three.

The `Field` class represents the JSON that shows the current state of the object today. The `History` represents the changelog that shows all the steps the issue has taken over time.

`issue.initialValue` checks `Field`, but it also goes back and checks `History`.

Reply:
@Anusien fixed